modules/space/rendering/renderableconstellationbounds.cpp

```diff
@@ -223,6 +189,11 @@ void RenderableConstellationBounds::render(const RenderData& data, RendererTasks
     }
     glBindVertexArray(0);
     _program->deactivate();
+
+    RenderableConstellation::render(data, tasks);
+}
+
+void RenderableConstellationBounds::update(const UpdateData& data) {
```

Comment:
Move closing bracket to this line?

Or how do we like to deal with empty functions?

Reply:
I think in this case we can omit the `update` function entirely.  The `update` function is not required for a Renderable